package.json

```diff
@@ -57,5 +64,15 @@
   "bugs": {
     "url": "https://github.com/The24thDS/stellaris-yaml/issues"
   },
-  "homepage": "https://github.com/The24thDS/stellaris-yaml/blob/main/README.md"
+  "homepage": "https://github.com/The24thDS/stellaris-yaml/blob/main/README.md",
+  "__metadata": {
+    "id": "3c607bba-2b85-49c0-afbf-f1e99e92b257",
+    "publisherId": "f63e6cf8-df88-48c9-9d40-9ab9bda726e7",
+    "publisherDisplayName": "The24thDS",
+    "targetPlatform": "undefined",
+    "updated": false,
+    "isPreReleaseVersion": false,
+    "installedTimestamp": 1687544745173,
+    "preRelease": false
+  }
```

Comment:
How did these get added? 😅

Reply:
I found that in the files coming from the VSCode extension market. I added it on a whim really, I have no idea if it's necessary or useful haha.

Reply:
We should probably remove it then. It looks like client side info since it has the installed time there